src/checkup-plugin-embroider/types/semver.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'semver';
```

Comment:
You should instead consider using `@types/semver`.

`yarn add @types/semver --dev`

Reply:
done